### ISSUE TITLE
Strong Naming Issue

### ISSUE THREAD

**Issue by RealDotNetDave** [aspnet/WebHooks#135]
None of the sites work for me. I keep getting this:

Could not load file or assembly 'Microsoft.AspNet.WebHooks.Common' or one of its dependencies. Strong name signature could not be verified.  The assembly may have been tampered with, or it was delay signed but not fully signed with the correct private key. (Exception from HRESULT: 0x80131045)

**Comment by HenrikFrystykNielsen** [comment]
Run build.cmd from the command line :)

**Comment by RealDotNetDave** [comment]
Great. Thanks.

**Comment by garora** [comment]
Sorry for interference. Is this issue related to is: [#92](https://github.com/aspnet/WebHooks/issues/92) ?

**Comment by HenrikFrystykNielsen** [comment]
Hi @garora, no it isn't. I haven't heard anything on that front :(

**Comment by garora** [comment]
Thanks for head's up. I am still waiting on that. It a quite long time now
:(

**Comment by dougbu** [comment]
This looks like a duplicate of #125. Both questions should be resolved following @HenrikFrystykNielsen's advice [here](https://github.com/aspnet/WebHooks/issues/135#issuecomment-298997999) or [here](https://github.com/aspnet/WebHooks/issues/125#issuecomment-287879987).

**Comment by dougbu** [comment]
Answered question is a duplicate of #125
